### ISSUE TITLE
Checkout email input field keeps `aria-invalid='true'` attribute and value after succesful validation

### ISSUE THREAD

**Issue by svossen** [magento/magento2#21573]
### Preconditions
I’m seeing this behavior on every Magento2 shop I tested, both in Firefox and Chrome. Even on _fresh_ Magento installations. The specifications for the situation of the attached screenshots are:
* Magento Open Source `2.3.0`
* Sample Data `100.3.*`
* macOS Mojave `10.14.2`
* PHP `7.1.25`
* MySQL `5.7.24`
* Firefox `66.0b12 (64-bit)`

### Steps to reproduce
1. Add product to cart
2. Go to checkout
3. Leave email input field empty
4. Validate form by going to the next step
5. Enter a valid email address
6. Validate again by blurring the input field or go to the next step

### Expected result
After successful validation, the `aria-invalid` attribute should be removed or the value should be set to `false`.

### Actual result
The checkout email input field keeps the `aria-invalid='true'` attribute and value after succesful email validation. The validation error message does gets hidden and the `mage-error` class does get removed and is replaced by `valid`. This situation will result in accessibility issues.

**Step 3:**
<img width="707" alt="screenshot 2019-03-05 at 11 06 25" src="https://user-images.githubusercontent.com/5097018/53797929-7dedd300-3f37-11e9-945a-fd7f6a880625.png">

**Step 4:**
<img width="719" alt="screenshot 2019-03-05 at 11 06 35" src="https://user-images.githubusercontent.com/5097018/53797938-81815a00-3f37-11e9-8414-3dbd8e77e767.png">

**Step 5 and 6:**
<img width="717" alt="screenshot 2019-03-05 at 11 06 50" src="https://user-images.githubusercontent.com/5097018/53797946-880fd180-3f37-11e9-802d-b05aa64429ed.png">

**Comment by magento-engcom-team** [comment]
Hi @svossen. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@svossen do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by svossen** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @svossen. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @svossen, here is your [Magento instance](https://i-21573-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21573-2-3-develop.instances.magento-community.engineering/admin](https://i-21573-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by svossen** [comment]
@magento-engcom-team I can confirm this occurs as well in the mentioned Magento instance.
<img width="933" alt="screenshot 2019-03-05 at 11 27 31" src="https://user-images.githubusercontent.com/5097018/53799112-feadce80-3f39-11e9-8725-a103d7d594b4.png">

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98561`, `MAGETWO-98562` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @Frodigo. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if you want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by darshan-khatri-16** [comment]
i am working on it #dmcindia1

**Comment by magento-engcom-team** [comment]
Hi @svossen. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. The fix was delivered into `magento/magento2:2.3-develop` branch(es).
Related commit(s):
 - [d218fd189b1b2b872d15e441eed9852f9ffcbfad](https://github.com/magento/magento2/commit/d218fd189b1b2b872d15e441eed9852f9ffcbfad)
 - [31bdcf568c5ac8e844e2e171c2f3e4351a8a402c](https://github.com/magento/magento2/commit/31bdcf568c5ac8e844e2e171c2f3e4351a8a402c)
 - [68ac344e2a016b8044910a604cc875acd52ae784](https://github.com/magento/magento2/commit/68ac344e2a016b8044910a604cc875acd52ae784)

The fix will be available with the upcoming `2.3.4` release.
